### PR TITLE
program_permission_required

### LINKED ISSUE #4981: Add hyperlink of FAQ to required table of repository-overview page
### Current Behavior
Currently there are some repos which are not accessible to everyone. This creates confusion for newcomer.
### Desired Behavior
It should explain why it is not accessible to everyone.
### Screenshots / Mockups
image
### Implementation
We can add hyperlink to that section which will lead to FAQ page where we can answer to the quiestion
---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
-  🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by Jat1nSood**
**Description**

This PR fixes #4981

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 49ea9554a56096fdb492eed0e5b720c3da992838 at: https://6525654966c52534721fd326--layer5.netlify.app

File: src/sections/Community/Handbook/faq.js [hunk 1]
```diff
@@ -16,61 +15,191 @@ const Faq = () => {
...
+              period in which you are given training and support, while you join
+              core team meetings to self-assess your desire to join the program
+              long-term and both self-assess and demonstrate your ability to be
+              a net positive contributor to the program. Program participants
+              are very well supported, learn and grow quickly. Their works are
+              often lauded on conference stages with bright spotlights placed on
+              their feats. These programs are intense, but amazing. Each a
+              participant is commits to maintain a specific level of daily
+              participation. If you are interested to explore, please inquire
+              with a <Link to="/community/community-managers">Community Manager</Link>, <Link to="/community/meshmates">MeshMate</Link>
```
Comment (Mohith234) [inline src/sections/Community/Handbook/faq.js:200]: - No space in the preview between `meshmate` and `or`
```suggestion
              with a <Link to="/community/community-managers">Community Manager</Link>, <Link to="/community/meshmates">MeshMate </Link>
```

File: src/sections/Community/Handbook/faq.js [hunk 2]
```diff
@@ -16,61 +15,191 @@ const Faq = () => {
...
+              to participate in one of these programs entails a two week tryout
+              period in which you are given training and support, while you join
+              core team meetings to self-assess your desire to join the program
+              long-term and both self-assess and demonstrate your ability to be
+              a net positive contributor to the program. Program participants
+              are very well supported, learn and grow quickly. Their works are
+              often lauded on conference stages with bright spotlights placed on
+              their feats. These programs are intense, but amazing. Each a
+              participant is commits to maintain a specific level of daily
+              participation. If you are interested to explore, please inquire
```
Comment (Mohith234) [inline src/sections/Community/Handbook/faq.js:199]: Couldn't understand this part. "Each a participant..."

**Review by Mohith234 — CHANGES_REQUESTED** [review]
I would like to add some suggestions

- Adding bullet points for the `Here’s how to get your invite:` part looks good instead of a single passage.
- Changing the wording of `Participation Required Programs` / `Participation Required Projects` to `"Program participation required" projects` sounds more intuitive in my opinion. Similar to as mentioned in the [repository overview](https://layer5.io/community/handbook/repository-overview) page. Thoughts @Jat1nSood?
- Some grammatical errors are present. Do proofread it once and fix them.

**Review by Gautam-04 — COMMENTED** [review]
As a contributor, you should consider providing screenshots of the changes you have made. This will make the task of maintainers easier and increase the chances of your PR getting merged faster.

File: src/sections/Community/Handbook/faq.js (hunk 1 above)
Comment (Jat1nSood) [reply src/sections/Community/Handbook/faq.js:200]: @Mohith234 Understood, I will do the needful.

File: src/sections/Community/Handbook/faq.js (hunk 2 above)
Comment (Jat1nSood) [reply src/sections/Community/Handbook/faq.js:199]: Yes, this should be " Each and every participant is committed"

**Comment by Jat1nSood** [comment]
> Member

@Gautam-04 Understood Gautam. Will take care of that next time.

**Comment by l5io** [comment]
🚀 Preview for commit 4f1dc328b3f872c456596178f4a2820180c7fc5c at: https://652951e7047f5543af32ce42--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 7dbdd925404199889bc652b43074716299665f9b at: https://652a283a1393ec70f96b2d4b--layer5.netlify.app

**Comment by abhijeetgauravm** [comment]
Hey @Jat1nSood  Let's discuss this on website call tomorrow at 5:30 PM IST / 7 AM Central time. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.lohhtewfwima).

**Comment by abhijeetgauravm** [comment]
Hey @Jat1nSood  Let's discuss this on website call tomorrow at 5:30 PM IST / 7 AM Central time. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.lohhtewfwima).

**Comment by leecalcote** [comment]
@Jat1nSood will you remove the unnecessary code changes? Most of what is changed is non-material to the issue at-hand.

**Comment by Jat1nSood** [comment]
> @Jat1nSood will you remove the unnecessary code changes? Most of what is changed is non-material to the issue at-hand.

Hi Lee, I am not sure what changes you are pointing to. Please elaborate. Do you want me to roll back to previous versions?

**Comment by Mohith234** [comment]
@Jat1nSood You might be using any one of the formatting extensions which is causing many changes for the other parts of the file but there are only just a few lines of change.

**Comment by Jat1nSood** [comment]
> @Jat1nSood You might be using any one of the formatting extensions which is causing many changes for the other parts of the file but there are only just a few lines of change.
> 
> P.S. Let's discuss this on website call today at 5:30 PM IST / 7 AM Central time. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit).

@Mohith234  I see, there are some extra lines and empty spaces which got added. Please confirm if those are the changes in question.  Also, I am very sorry, but it is difficult for me to join at that time. I work full time and I can't join during office hours.

**Comment by Mohith234** [comment]
@Jat1nSood Yes, due to the formatting extension there are many lines of changes happening but there is no need of it so please remove those formatting spaces.

**Review by leecalcote — APPROVED** [review]
Thanks for seeing this through, gents.

**Comment by leecalcote** [comment]
Uh-oh, we have a merge conflict. @Jat1nSood will you investigate and reconcile?

**Comment by Jat1nSood** [comment]
@Mohith234 @abhijeetgauravm  Guys any idea why these checks are failing?

**Review by leecalcote — CHANGES_REQUESTED** [review]
Please undo all changes not pertinent to the issue at-hand.

**Comment by l5io** [comment]
🚀 Preview for commit 809cf05f4b7453000abc7f5e9724cc0bb3572ee0 at: https://656ebad171505c168ade0760--layer5.netlify.app

File: package-lock.json
Comment (leecalcote) [inline package-lock.json:1]: Please exclude any changes to this file.

File: src/sections/Community/Handbook/faq.js [hunk 3]
```diff
@@ -18,58 +17,200 @@ const Faq = () => {
           <div className="content">
             <h2 className="heading-top">General FAQs:</h2><br />
             <h4>Q: Are Layer5’s solutions open source?</h4>
-            <p>A: Yes, all Layer5 projects are licensed under Apache V2. Layer5 also offers extensions to the Meshery project in which Enterprise-centric functionality is commercially offered and supported.</p>
+            <p>
```
Comment (leecalcote) [inline src/sections/Community/Handbook/faq.js:20]: This change and others like it are unwanted. Please revert these.

**Review by leecalcote — CHANGES_REQUESTED** [review]

**Comment by Jat1nSood** [comment]
Hi @leecalcote , I am not sure what happened here.

**Pull request closed without merging**